### PR TITLE
[DOCS-3408] Clarify where DD_RUM.addAction comes from

### PR DESCRIPTION
**Pull request by jonnyboyC**
### What does this PR do?
This PR shows an explicit import in a code snippet that was previously missing.

### Motivation
From an initial read through of the docs it wasn't clear what `DD_RUM` was or where it came from.  Poking around in the TS types it seems this was actually `datadogRum` from '@datadog/browser-rum'

### Preview
I'm on a team using DD so I don't believe I have access to this.

### Additional Notes
N/A

---

### Reviewer checklist
- [x] Review the changed files.
- [x] Review the URLs listed in the [Preview](#preview) section.
- [x] Check images for PII
- [x] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by jtappa** [comment]
Hi @jonnyboyC , thanks for your contribution! I've created a Jira card for this issue so that it can be verified for you.

File: content/en/real_user_monitoring/guide/send-rum-custom-actions.md [hunk 1]
```diff
@@ -16,8 +16,10 @@ Real User Monitoring [automatically collects actions][1] on your web application
 To create a new RUM action, use the `addAction` API. Give your action a name and then attach context attributes in the form of a JavaScript object. In the following example, a `checkout` action is created with details about the user cart when the user clicks on the checkout button.
 
 ```javascript
+import { datadogRum } from '@datadog/browser-rum';
+
 function onCheckoutButtonClick(cart) {
-    DD_RUM.addAction('checkout', {
+    datadogRum.addAction('checkout', {
```
Comment (bcaudan) [inline content/en/real_user_monitoring/guide/send-rum-custom-actions.md:22]: As detailed in the [action doc](https://docs.datadoghq.com/real_user_monitoring/browser/tracking_user_actions/?tab=npm#custom-actions), the syntax depends on your setup (NPM / CDN async / CDN sync).
Adding the different versions in this guide could be a nice improvement.

**Review by alai97 — APPROVED** [review]

File: content/en/real_user_monitoring/guide/send-rum-custom-actions.md [hunk 2]
```diff
@@ -4,45 +4,90 @@ kind: guide
...
+DD_RUM.onReady(function() {
+    DD_RUM.addAction('<NAME>', '<JSON_OBJECT>');
+})
+
+function onCheckoutButtonClick(cart) {
+    DD_RUM.addAction('checkout', {
+        'value': cart.value, // for example, 42.12
+        'items': cart.items, // for example, ['tomato', 'strawberries']
+    })
+}
```
Comment (bcaudan) [inline content/en/real_user_monitoring/guide/send-rum-custom-actions.md:49]: **💬 suggestion:** 
- example only to be consistent with npm tab
- ensure to wrap api call in `onReady` callback

```suggestion
function onCheckoutButtonClick(cart) {
    DD_RUM.onReady(function() {
        DD_RUM.addAction('checkout', {
            'value': cart.value, // for example, 42.12
            'items': cart.items, // for example, ['tomato', 'strawberries']
        })
    })    
}
```

File: content/en/real_user_monitoring/guide/send-rum-custom-actions.md [hunk 3]
```diff
@@ -4,45 +4,90 @@ kind: guide
...
 ```javascript
+window.DD_RUM && DD_RUM.addAction('<NAME>', '<JSON_OBJECT>');
+
 function onCheckoutButtonClick(cart) {
     DD_RUM.addAction('checkout', {
         'value': cart.value, // for example, 42.12
-        'items': cart.items, // efor example, ['tomato', 'strawberries']
+        'items': cart.items, // for example, ['tomato', 'strawberries']
     })
 }
```
Comment (bcaudan) [inline content/en/real_user_monitoring/guide/send-rum-custom-actions.md:62]: **💬 suggestion:** ‏
- example only to be consistent with npm tab
- ensure to check for DD_RUM existence before api call

```javascript
function onCheckoutButtonClick(cart) {
    window.DD_RUM && DD_RUM.addAction('checkout', {
        'value': cart.value, // for example, 42.12
        'items': cart.items, // for example, ['tomato', 'strawberries']
    })
}
```

**Review by bcaudan — APPROVED** [review]
LGTM!

**Pull request merged**